### PR TITLE
Nicos/audio

### PR DESCRIPTION
**Pull request by nicolasgustafsson**
Implemented SoLoud audio library

### CODE REVIEW

File: WonderMake/Program/Program.cpp [hunk 1]
```diff
@@ -33,6 +33,8 @@ void Program::StartNewFrame()
 	myCameraPtr->Update();
 
 	myRendererPtr->StartFrame();
+
+	myAudioManagerPtr->Update();
```
Comment (Devilhar) [inline WonderMake/Program/Program.cpp:37]: Shouldn't this be in the Update function instead?

File: WonderMakeLauncher/WonderMakeLauncher.vcxproj.filters [hunk 2]
```diff
@@ -3,12 +3,56 @@
   <ItemGroup>
     <ClCompile Include="Main.cpp" />
     <ClCompile Include="pch.cpp" />
-    <ClCompile Include="Registry\*.cpp" />
+    <ClCompile Include="Registry\Register_ActionFunctionality.cpp" />
```
Comment (Devilhar) [inline WonderMakeLauncher/WonderMakeLauncher.vcxproj.filters:6]: Please remove these.

File: Game/Game.vcxproj [hunk 3]
```diff
@@ -75,31 +75,31 @@
     <OutDir>$(SolutionDir)Bin\</OutDir>
     <IntDir>$(SolutionDir)Intermediate\$(ProjectName)\$(Platform)\$(Configuration)\</IntDir>
     <TargetName>$(ProjectName)_$(Platform)_$(Configuration)</TargetName>
-    <IncludePath>$(SolutionDir)$(ProjectName);$(SolutionDir)External\glad\Inc;$(SolutionDir)External\glfw-3.3\Inc;$(SolutionDir)External\Imgui-1.68 WIP\Inc;$(SolutionDir)External\json-3.5.0\Inc;$(SolutionDir)External\KHR\Inc;$(SolutionDir)External\MetaStuff\Inc;$(IncludePath)</IncludePath>
+    <IncludePath>$(SolutionDir)$(ProjectName);$(SolutionDir)External\glad\Inc;$(SolutionDir)External\glfw-3.3\Inc;$(SolutionDir)External\Imgui-1.68 WIP\Inc;$(SolutionDir)External\json-3.5.0\Inc;$(SolutionDir)External\SoLoud\Inc;$(SolutionDir)External\KHR\Inc;$(SolutionDir)External\MetaStuff\Inc;$(IncludePath)</IncludePath>
```
[the remark this answers is not in the document]
Comment (nicolasgustafsson) [reply Game/Game.vcxproj]: Fixed

File: WonderMake/Audio/AudioFile.cpp [hunk 4]
```diff
@@ -0,0 +1,12 @@
+#include "pch.h"
+#include "AudioFile.h"
+
+AudioFile::AudioFile(const std::filesystem::path& aPath)
+{
+	mySource.load(aPath.string().c_str());
```
[the remark this answers is not in the document]
Comment (nicolasgustafsson) [reply WonderMake/Audio/AudioFile.cpp:6]: No unload function, it gets removed in the destructor :)

File: WonderMake/Program/Program.cpp (hunk 1 above)
Comment (nicolasgustafsson) [reply WonderMake/Program/Program.cpp]: Fixed

File: WonderMakeLauncher/WonderMakeLauncher.vcxproj.filters (hunk 2 above)
Comment (nicolasgustafsson) [reply WonderMakeLauncher/WonderMakeLauncher.vcxproj.filters]: Fixed

**Pull request merged**